quickstart/src/main/java/com/twilio/video/quickstart/activity/VideoActivity.java

```diff
@@ -521,13 +525,13 @@ private void showAudioDevices() {
     private void updateAudioDeviceIcon(AudioDevice selectedAudioDevice) {
```

Comment:
Looks like you forgot to add the when expression here. I think it is a lot easier to read this way and is less code.
```
private fun updateAudioDeviceIcon(selectedAudioDevice: AudioDevice?) {
    val audioDeviceMenuIcon = when(selectedAudioDevice) {
        is BluetoothHeadset -> R.drawable.ic_bluetooth_white_24dp
        is WiredHeadset -> R.drawable.ic_headset_mic_white_24dp
        is Speakerphone -> R.drawable.ic_volume_up_white_24dp
        else -> R.drawable.ic_phonelink_ring_white_24dp
    }

    audioDeviceMenuItem.setIcon(audioDeviceMenuIcon)
}
```